eclair-core/src/main/scala/fr/acinq/eclair/Features.scala

```diff
@@ -36,12 +35,13 @@ object Features {
   val CHANNEL_RANGE_QUERIES_BIT_MANDATORY = 6
   val CHANNEL_RANGE_QUERIES_BIT_OPTIONAL = 7
 
+  val OPTION_VARIABLE_LENGTH_ONION_MANDATORY = 0
```

Comment:
I guess that's still TBD.

Reply:
Yes exactly, there are two small moving parts that are pending comments from Christian on the spec, I detailed that in the description of this PR.